cnapy/gui_elements/central_widget.py

```diff
@@ -203,7 +203,7 @@ def add_map(self):
         m = CnaMap(name)
 
         self.appdata.project.maps[name] = m
-        mmap = MapView(self.appdata, name)
+        mmap = MapView(self.appdata, self, name)
```

Comment:
Maybe just give `CentralWidget` to `Mapview`?

Reply:
Yes, in this case self is actually the CentralWidget.

Reply:
Yes. I mean central widget has the appdata. In map view you can just use the appdata from the central widget and don't need to have appdata as argument in the constructor. 
Also this way it would be clear that the map view always uses the appdata from the central widget.

Reply:
Oh, I wasn't aware of that. I saw several constructors where both parameters are being passed, so I just followed that.